test_downstream.py randomly fails

I've noticed the following failure that happens rarely:

```
======================================================================
FAIL: test_run_alpha_rarefaction (__main__.DownstreamWorkflowTests)
run_alpha_rarefaction generates expected results
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/workspace/qiime-github-pr/qiime_software/qiime-1.6.0-repository-1e9a5129/tests/test_workflow/test_downstream.py", line 252, in test_run_alpha_rarefaction
    self.assertTrue(a['feces,L_palm'][1] < 0.15)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 8 tests in 183.431s

FAILED (failures=1)
```

--- a/tests/test_workflow/test_downstream.py
+++ b/tests/test_workflow/test_downstream.py
@@ -54,12 +54,6 @@ class DownstreamWorkflowTests(TestCase):
         
         self.qiime_config = load_qiime_config()
         self.params = parse_qiime_parameters(params_f1)
-        
-        # set the maximum p-value we'll allow for a t-test of 
-        # fecal alpha diversities to palm alpha diversities. this gets 
-        # used in several places, and is often less than 0.15, but 
-        # occasionally higher, so can cause stochastic failures.
-        self.feces_palm_alpha_max_p = 0.20
 
         # suppress stderr during tests (one of the systems calls in the 
         # workflow prints a warning, and we can't suppress that warning with 
@@ -255,10 +249,10 @@ class DownstreamWorkflowTests(TestCase):
                                       'SampleType', 
                                       18,
                                       test_type='parametric')
-        feces_palm_p = a['feces,L_palm'][1]
-        self.assertTrue(feces_palm_p <= self.feces_palm_alpha_max_p, 
-         "p-value too high: %1.3f, but prefer <= %1.3f" % \
-         (feces_palm_p, self.feces_palm_alpha_max_p))
+        feces_palm_t = a['feces,L_palm'][0]
+        self.assertTrue(feces_palm_t < 0, 
+         "t-statistic too high: %1.3f, but should be less than 0"\
+          % feces_palm_t)
         
         # check that final output files have non-zero size
         self.assertTrue(getsize(html_fp) > 0)
@@ -306,10 +300,10 @@ class DownstreamWorkflowTests(TestCase):
                                       'SampleType', 
                                       18,
                                       test_type='parametric')
-        feces_palm_p = a['feces,L_palm'][1]
-        self.assertTrue(feces_palm_p <= self.feces_palm_alpha_max_p, 
-         "p-value too high: %1.3f, but prefer <= %1.3f" % \
-         (feces_palm_p, self.feces_palm_alpha_max_p))
+        feces_palm_t = a['feces,L_palm'][0]
+        self.assertTrue(feces_palm_t < 0, 
+         "t-statistic too high: %1.3f, but should be less than 0"\
+          % feces_palm_t)
         
         # check that final output files have non-zero size
         self.assertTrue(getsize(html_fp_stderr) > 0)
@@ -354,10 +348,10 @@ class DownstreamWorkflowTests(TestCase):
                                       'SampleType', 
                                       18,
                                       test_type='parametric')
-        feces_palm_p = a['feces,L_palm'][1]
-        self.assertTrue(feces_palm_p <= self.feces_palm_alpha_max_p, 
-         "p-value too high: %1.3f, but prefer <= %1.3f" % \
-         (feces_palm_p, self.feces_palm_alpha_max_p))
+        feces_palm_t = a['feces,L_palm'][0]
+        self.assertTrue(feces_palm_t < 0, 
+         "t-statistic too high: %1.3f, but should be less than 0"\
+          % feces_palm_t)
         
         # check that final output files have non-zero size
         self.assertTrue(getsize(html_fp) > 0)

